src/Service.Tests/CosmosTests/TestBase.cs

```diff
@@ -42,20 +45,33 @@ public class TestBase
     character: Character,
     age : Int,
     dimension : String,
+    earth: Earth,
     stars: [Star],
     moons: [Moon],
     tags: [String!]
 }
 
 type Star @model(name:""StarAlias"") {
     id : ID,
-    name : String
+    name : String,
+    tag: Tag
 }
 
+type Tag @model(name:""TagAlias"") {
+    id : ID,
+    name : String
+} 
+
 type Moon @model(name:""Moon"") @authorize(policy: ""Crater"") {
```

Comment:
Shouldn't this be `roles`?

Reply:
No, this shouldn't be changed here. There is a specific test in CosmosTests.QueryTests called: `GetWithInvalidAuthorizationPolicyInSchema()` which validates the behavior in `GraphQLAuthorizationHandler` to deny a request (HTTP 403) for GraphQL schemas which use the authorize directive denoting a policy `@authorize(policy:"")`. Policy is a Asp.net Authorization feature we do not use. That test was added to validate changes that I made to `GraphQLAuthorizationHandler` a little while back.

Reply:
good to know, thanks!